docs/configuration/monitoring.md

```diff
@@ -1,43 +1,140 @@
 # Monitoring
 
-In older versions, Codacy was monitored using an application we build
-in-house called "Crow". Going forward, we are deprecating Crow and
-setting up "Prometheus style" metrics in all our components that
-can be accessed using the Prometheus + Grafana ecosystem.
+Currently two monitoring solutions are supported:
 
-Initially Crow will still be bundled in Codacy helm chart, until
-we have all functionality available on Grafana.
+* [**Crow**](##Setting-up-monitoring-using-Crow) - A simple, lightweight, and builtin monitoring solution.
+* [**Prometheus+Grafana+Loki**](##Setting-up-monitoring-using-Grafana,-Prometheus,-and-Loki) - A more comprehensive third-party monitoring solution, for more advanced usage.
 
-## Setting up monitoring using Grafana & Prometheus
+## Setting up monitoring using Crow
+
+Crow is a visualization tool that displays information about the projects and jobs
+that are pending for analysis, as well as the ones running in the Codacy platform.
```

Comment:
Just to clarify, I interpret this as "Crow displays information about the **projects pending analysis** and the **jobs currently running** on Codacy". Is this correct?

Reply:
I barely touched the Crow documentation. I just moved it to the top of the document. It was changed just a week ago so I assumed it was thoroughly reviewed then. These sentences seem equivalent, but I can change it nonetheless.